Assets/Scripts/Framework/SceneSwitcher.cs

```diff
@@ -18,7 +22,31 @@ private void Awake()
         /// Load the scene that is set (sceneToLoad property).
         /// </summary>
         public void LoadScene() => SceneManager.LoadScene(sceneToLoad);
+        
+        /// <summary>
+        /// Loads the scene that is set Asynchronously (sceneToLoad property).
+        /// </summary>S
+        IEnumerator LoadSceneAsync()
+        {
+            AsyncOperation operation = SceneManager.LoadSceneAsync(sceneToLoad);
+
+            while (!operation.isDone)
+            {
+                Progress = Mathf.Clamp01(operation.progress / 0.9f);
```

Comment:
Waarom delen we dit door 0.9 en als dit moet kan het beter een const zijn met een naam die verteld wat het is.

Reply:
Dit delen we door 0.9 omdat een async operation van 0.0 naar 0.9 gaat. Dus om een laadbalk van 0 > 1 te laten gaan moet je t delen door 0.9